src/proposals/20240306/SparkEthereum_20240306.t.sol

```diff
@@ -0,0 +1,447 @@
+            gap:              100,
+            increaseCooldown: 12 hours
+        });
+
+        /************/
+        /*** sDAI ***/
+        /************/
+
+        _assertSupplyCapConfig({
+            asset:            SDAI,
+            max:              1_000_000_000,
+            gap:              50_000_000,
+            increaseCooldown: 12 hours
+        });
+
+        _assertBorrowCapConfigNotSet(SDAI);
+
+        /************/
+        /*** USDC ***/
+        /************/
+
+        _assertSupplyCapConfigNotSet(USDC);
+
+        _assertBorrowCapConfig({
+            asset:            USDC,
+            max:              57_000_000,
+            gap:              6_000_000,
+            increaseCooldown: 12 hours
+        });
+
+        /************/
+        /*** USDT ***/
+        /************/
+
+        _assertSupplyCapConfigNotSet(USDT);
+
+        _assertBorrowCapConfig({
+            asset:            USDT,
+            max:              28_500_000,
+            gap:              3_000_000,
+            increaseCooldown: 12 hours
+        });
+
+        /************/
+        /*** WBTC ***/
+        /************/
+
+        _assertSupplyCapConfig({
+            asset:            WBTC,
+            max:              5_000,
+            gap:              500,
+            increaseCooldown: 12 hours
+        });
+
+        _assertBorrowCapConfig({
+            asset:            WBTC,
+            max:              2_000,
+            gap:              100,
+            increaseCooldown: 12 hours
+        });
+
+        /************/
+        /*** WETH ***/
+        /************/
+
+        _assertSupplyCapConfig({
+            asset:            WETH,
+            max:              2_000_000,
+            gap:              150_000,
+            increaseCooldown: 12 hours
+        });
+
+        _assertBorrowCapConfig({
+            asset:            WETH,
+            max:              1_000_000,
+            gap:              10_000,
+            increaseCooldown: 12 hours
+        });
+
+        /**************/
+        /*** wstETH ***/
+        /**************/
+
+        _assertSupplyCapConfig({
+            asset:            WSTETH,
+            max:              1_200_000,
+            gap:              50_000,
+            increaseCooldown: 12 hours
+        });
+
+        _assertBorrowCapConfig({
+            asset:            WSTETH,
+            max:              3_000,
+            gap:              100,
+            increaseCooldown: 12 hours
+        });
+    }
+
+    function testCapAutomatorCapUpdates() public {
+        GovHelpers.executePayload(vm, payload, executor);
+
+        _assertAutomatedCapsUpdate(RETH);
+        _assertAutomatedCapsUpdate(SDAI);
+        _assertAutomatedCapsUpdate(USDC);
+        _assertAutomatedCapsUpdate(USDT);
+        _assertAutomatedCapsUpdate(WBTC);
+        _assertAutomatedCapsUpdate(WETH);
+        _assertAutomatedCapsUpdate(WSTETH);
```

Comment:
I would also just run over each asset with hardcoded values to make sure the results make sense.

Ex) `assertEq(weth.configuration.getSupplyCap(), 388_000)` (or whatever it is actually 238k existing + 150k gap)